rds-dc-with-one-vm-domainjoin/Readme.md

```diff
@@ -0,0 +1,29 @@
+# Domain join sample
```

Comment:
Perhaps a better folder name would be 201-domain-join or something along 201 lines since this is intended to be a sample and I can see this being fairly common to get started on domain join scenarios